inc/cubd.h

```diff
@@ -38,41 +38,67 @@
 /*************************************************/
 /* macros ****************************************/
 /*************************************************/
-# define MAP "Error\nCan't open map. Maybe file does not exist.\n"
+# define MAP "Error\nCan't open map. File may be corrupted.\n"
 # define NAME "Error\nMap name is invalid\n"
 # define MALLOC "Error\nMalloc Failure\n"
 # define FILE "Error\nElements may be incorrect. Check map conditions.\n"
+# define SPRITE "Error\nCould not find/load sprites."
+# define RGB1 "Error\nRGB (ceiling) values: value must be between 0-255\n"
+# define RGB2 "Error\nRGB (floor) values: value must be between 0-255\n"
+# define PLAYER "Error\nMust have one player character(N, W, S, or E)\n"
+# define BRK_MAP "Error\nMap is broken ):\n"
+# define CLOSE "Error\nMap is not walled correctly ):<\n"
 
 # define SUCCESS 0
 # define FAILURE 1
+# define SUFFER 65
 
 /*************************************************/
 /* usage macros **********************************/
 /*************************************************/
-# define USAGE1 "Welcome to cub3d "
-# define USAGE2 "and you already have committed your first mistake."
+# define USAGE1 "\nWelcome to cub3d "
+# define USAGE2 "and you already have committed a folly."
 # define USAGE3 " Follow the usage below:\n"
 # define USAGE4 "1. map must have the suffix '.cub';\n"
 # define USAGE5 "2. In clt just execute the program and map;\n"
-# define USAGE6 "3. Map must contain all elements (sprites with "
-# define USAGE7 "cardinal orientation, proper walled map, and "
+# define USAGE6 "3. Map must contain all elements (sprites with \n"
+# define USAGE7 "cardinal orientation, proper walled map, and \n"
 # define USAGE8 "RGB notated instructions [numerical value]);\n\n"
 # define USAGE9 "This game has no exit for now, so use ESC to "
 # define USAGE10 "finish the game (:\n\n"
-# define USAGE11 "By: fdessoy- | araveala\n"
+# define USAGE11 "By: fdessoy- && araveala\n"
 
 /*************************************************/
 /* structs ***************************************/
 /*************************************************/
```

Comment:
different variable names to mine, am now using yours, ppos, map_width and map _length (height might be more of a clear name)

Reply:
I guess we talked about variable names last meeting and I think you managed to change them using the VS code global search. Closing conversation for final merge.